test/end-to-end.js

```diff
@@ -97,6 +98,20 @@ describe('end-to-end', function() {
         .expect('Access-Control-Allow-Origin',  'http://example.com')
         .expect(200, done);
     });
+    
+    it('provides security headers for all URLs ', function(done){
+      request(app).get('/')
+       .expect('X-frame-options', 'DENY')
+       .expect('x-xss-protection', '1; mode=block')
+       .expect('x-content-type-options', 'nosniff')
+       .expect('x-download-options', 'noopen')
+       .expect(function(res){
+          var headers = res.headers;
+          console.log(headers);
```

Comment:
@bajtos : Grrrr... I forgot to remove `console.log`, it was for my own reference only...
I believe, `headers.should.not.have.property` throws enough error details, I will remove the log part.